eos/rare-b-decays/nonlocal-formfactors-p-to-v.cc

```diff
@@ -473,6 +487,32 @@ namespace eos
                     return H_residue_psi2s(phiParam, alpha_0, alpha_1, alpha_2);
                 }
 
+                virtual complex<double> normalized_perp(const double & q2) const
+                {
+                    const double m_V2  = pow(m_V, 2);
+                    const double m_B2  = pow(m_B, 2);
+                    const double lambda = pow(m_B2, 2) + pow(m_V2, 2) + pow(q2, 2) - 2*m_B2*m_V2 - 2*m_B2*q2 - 2*q2*m_V2;
```

Comment:
Why not use ``eos::lambda``, which is defined in ``eos/utils/kinematic.hh``? :smile: